rewrite-groovy/src/test/java/org/openrewrite/groovy/tree/MethodInvocationTest.java

```diff
@@ -35,6 +35,11 @@ void gradle() {
               repositories {
                   mavenCentral()
               }
+              
+              copySpec {
+                  from { 'src/main/webapp' } { exclude "**/*.jpg" }
+                  rename '(.+)-staging(.+)', '$1$2'
+              }
```

Comment:
The first test in our test classes usually serves as a minimal happy path test; to be copied for subsequent tests to test specific elements. As such I think it's not desirable to add this block to the topmost test, but rather add a second test, appropriately annotated with `@Issue` that merely adds this block and ensures we can still parse the block.